src/lwc/geFormFieldPicklist/geFormFieldPicklist.js

```diff
@@ -13,14 +15,20 @@ export default class GeFormFieldPicklist extends LightningElement {
     @track _objectDescribeInfo;
     @track picklistValues;
     @track defaultRecordTypeId;
+
     _recordTypeId;
+    CUSTOM_LABELS = GeLabelService.CUSTOM_LABELS;
 
     @wire(getPicklistValues, {
         fieldApiName: '$fullFieldApiName',
         recordTypeId: '$_recordTypeId' })
     wiredPicklistValues({error, data}) {
         if(data) {
-            this.picklistValues = data.values;
+            let valueNone = {
+                label: this.CUSTOM_LABELS.commonLabelNone,
+                value: this.CUSTOM_LABELS.commonLabelNone
```

Comment:
Would making the value portion of this picklist option be either empty string or null be better?

Reply:
@bdvorachek yeah you might think so but that `value` property seems to actually control what is displayed in the field. If I set that to `null` the value in the field loads as blank.

Reply:
@bdvorachek you would think so but making the `value` property null seems to be tied to the display value. When you make the `value` null it removes the displayed value